### DISCUSSION TITLE
JSCAD (previously known as OpenJSCAD.org)

### DISCUSSION THREAD

**Discussion by platypii** [jscad/OpenJSCAD.org#1026] [General]
It seems like there is a slow migration away from OpenJSCAD.org and toward simply "JSCAD".

This makes sense to me for a number of reasons:

1. The domain openjscad.org is dead, so the repo name is rather misleading.
2. The README says it is "formerly known as" openjscad.org
3. JSCAD matches the npm package names
4. On the application it just says "JSCAD"
5. It's a library not just a website
6. JSCAD is a cleaner, simpler name

Curious if there are thoughts on taking the leap and renaming the repo on github to `jscad/jscad`? One thing to note is that github will automatically forward to the new URL. So if users visit an old link to https://github.com/jscad/OpenJSCAD.org it will still work.

**Comment by hrgdavor** [comment]
sounds reasonable :)

**Reply by gilboonet** [reply]
Anything stable would be appreciated, I have lots of shared designs that are now dead links, I try to correct them whenever possible, but it's not always possible. Anyway JSCAD design sharing is not something common.
